include/deal.II/grid/cell_id.h

```diff
@@ -259,11 +252,15 @@ template <class Archive>
 void
 CellId::serialize(Archive &ar, const unsigned int /*version*/)
 {
+  unsigned int n_child_indices = child_indices.size();
```

Comment:
I've added this line for compatibility reasons.

Reply:
No need for this. We do not guarantee compatibility regarding serialization.